bitnami/argo-cd/values.yaml

```diff
@@ -0,0 +1,2048 @@
+  ##
+  podSecurityContext:
+    enabled: true
+    fsGroup: 1001
+  ## Configure Container Security Context
+  ## ref: https://kubernetes.io/docs/tasks/configure-pod-container/security-context/#set-the-security-context-for-a-pod
+  ## @param server.containerSecurityContext.enabled Enabled Argo CD server containers' Security Context
+  ## @param server.containerSecurityContext.runAsUser Set Argo CD server containers' Security Context runAsUser
+  ##
+  containerSecurityContext:
+    enabled: true
+    runAsUser: 1001
+
+  ## Argo CD server deployment autoscaling
+  ## @param server.autoscaling.enabled Enable Argo CD server deployment autoscaling
+  ## @param server.autoscaling.minReplicas Argo CD server deployment autoscaling minimum number of replicas
+  ## @param server.autoscaling.maxReplicas Argo CD server deployment autoscaling maximum number of replicas
+  ## @param server.autoscaling.targetCPU Argo CD server deployment autoscaling target CPU percentage
+  ## @param server.autoscaling.targetMemory Argo CD server deployment autoscaling target CPU memory
+  ##
+  autoscaling:
+    enabled: false
+    minReplicas: 1
+    maxReplicas: 5
+    targetCPU: 50
+    targetMemory: 50
+
+  ## Redirect all request to https
+  ## @param server.insecure Disable HTTPS redirection for Argo CD server
+  ##
+  insecure: false
+
+  ## @param server.logFormat ArgoCD server logs format. Options: [text, json]
+  ##
+  logFormat: text
+  ## @param server.logLevel ArgoCD server logs level
+  ##
+  logLevel: info
+
+  ## Argo CD server enable config
+  ## @param server.configEnabled Enable Argo CD server config
+  ## Ref: https://github.com/argoproj/argo-cd/blob/master/docs/operator-manual/argocd-cm.yaml
+  ##
+  configEnabled: true
+
+  ## Argo CD server config. This object will be directly rendered
+  ## Ref: https://argoproj.github.io/argo-cd/operator-manual/user-management/
+  ## E.g:
+  ## repositories: |
+  ##   - url: git@github.com:group/repo.git
+  ##     sshPrivateKeySecret:
+  ##       name: secret-name
+  ##       key: sshPrivateKey
+  ##   - type: helm
+  ##     url: https://charts.helm.sh/stable
+  ##     name: stable
+  ##   - type: helm
+  ##     url: https://argoproj.github.io/argo-helm
+  ##     name: argo
+  ## oidc.config: |
+  ##   name: AzureAD
+  ##   issuer: https://login.microsoftonline.com/TENANT_ID/v2.0
+  ##   clientID: CLIENT_ID
+  ##   clientSecret: $oidc.azuread.clientSecret
+  ##   requestedIDTokenClaims:
+  ##     groups:
+  ##       essential: true
+  ##   requestedScopes:
+  ##     - openid
+  ##     - profile
+  ##     - email
+  ## dex.config: |
+  ##    connectors:
+  ##      # GitHub example
+  ##      - type: github
+  ##        id: github
+  ##        name: GitHub
+  ##        config:
+  ##          clientID: aabbccddeeff00112233
+  ##          clientSecret: $dex.github.clientSecret
+  ##          orgs:
+  ##          - name: your-github-org
+  config:
```

Comment:
If this is a configuration file, I'd turn it to string to make sure this is the configuration file from ArgoCD and not actual values from the chart

```
config: |
```